gammapy/estimators/points/sed.py

```diff
@@ -56,32 +56,36 @@ class FluxPointsEstimator(FluxEstimator, parallel.ParallelMixin):
     energy_edges : list of `~astropy.units.Quantity`, optional
         Edges of the flux points energy bins. The resulting bin edges won't be exactly equal to the input ones,
         but rather the closest values to the energy axis edges of the parent dataset.
-        Default is None: apply the estimator in each energy bin of the parent dataset.
-        For further explanation see :ref:`estimators`.
-    fit : `Fit`
-        Fit instance specifying the backend and fit options.
-    reoptimize : bool
-        If True the free parameters of the other models are fitted in each bin independently,
+        Default is [1, 10] TeV.
+    fit : `Fit`, optional
+        Fit instance specifying the backend and fit options. Default is None, so "scipy" is used.
```

Comment:
Technically the default `Fit` relies on the scipy backend  only for confidence (i.e. for `errn-errp` and `ul` options). This is because the scipy minimizer seems to behave a bit better for UL calculations.

To me this is internal cuisine and does not really need to be explained in the argument description.